workflows/README.adoc

```diff
@@ -0,0 +1,9 @@
+= Github Workflows
+
+== Reusable workflows
+
+These workflows can be used as-is without modifications to enable certain functionality
+
+== Workflow templates
+
+These workflows are commonly used but might require some per-project tweaks
```

Comment:
== Reusable workflows
vs
== Workflow templates

how they differ? level 2 titles both, they confuse me.

Reply:
Thought about "Reusable workflow" working with no or only external edits and "Workflow templates" that will give a copyable skeleton that gets edited. Might as well remove those and just have single README that says whatever needs to be done and with what file